redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
```

Comment:
I like them for overriding methods as it allows super-methods to process further arguments that we don't care about.

The hard coded args might make sense in end-product code, but in library code we need to support people with different versions of other libraries.